pages/wg/dei.md

```diff
@@ -28,9 +28,14 @@ The US-RSE DEI working group's first large initiative was to create a US-RSE DEI
 
 ## DEI Speaker Series
 
-The US-RSE DEI working group invites DEI speakers to present on DEI topics. In 2021, we hosted [Dr. Cherri Pancake]({{ site.url }}/events/2021/2021-03-dei-speaker-series/) as she presented her talk, "Can We Change the Face of Computing?" and [Joanne Esch and Andrea Bowen-Jones]({{ site.url }}/events/2021/2021-07-dei-speaker-series-ncwit) of the National Center for Women & Information Technology.
+The US-RSE DEI working group invites DEI speakers to present on DEI topics.
+A playlist with talks from all the spekaers to date
+can be found on the US-RSE YouTube channel here:  
+<https://www.youtube.com/playlist?list=PLMgG8jy8w-8lUu8Uflt1WCsI8yzEEVybS>
```

Comment:
Might we want to make this a link for the text "US-RSE YouTube channel" or similar?

Reply:
I hear you and thought about that but personally I find it very hard to see the links in the current site color scheme + styling, and I'm afraid people will miss it. So I thought it was better to just make it an explicit link

Reply:
Force pushed to fix the "spekars" typo

Reply:
Could be we need to rethink the colors some then!

Reply:
Agreed, as in #618 :slightly_smiling_face: 

Do you need me to hide this external link before we merge?

Reply:
Ah nvm, saw you merged, thank you! :tada: :rocket:

Reply:
No, we can adjust after if anyone feels strongly otherwise, no need to hold this up.

Reply:
:+1: understood, appreciate it